SECURITY.md

```diff
@@ -261,11 +261,31 @@ distribution should:
       marketing copy, etc.) that it is built on top of Envoy. E.g.,
       "SuperAwesomeCloudProvider's Envoy as a Service (EaaS)". An infrastructure
       service that uses Envoy for a product but does not publicly say they are
-      using Envoy does not qualify. This is essentially IaaS or PaaS, if you use
-      Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not
-      qualify.
-2. Have a user or customer base not limited to your own organization. We will use the size
-   of the user or customer base as part of the criteria to determine
+      using Envoy does not *generally* qualify (see option 3 that follows). This is essentially IaaS
+      or PaaS. If you use Envoy to support a SaaS, e.g. "SuperAwesomeCatVideoService", this does not
+      *generally* qualify.
+
+   OR
+
+   3. An end user of Envoy that satisfies the following requirements:
```

Comment:
Why do we skip 2 here?

Reply:
It's not skipped, just indented. Check the rendered output.